account/account-service.test.js

```diff
@@ -54,4 +56,13 @@ describe('Account service', () => {
             });
         });
     });
+    describe('isAuthenticated', () => {
+        it('Should return authenticated state', () => {
+            const accountService = createAccountService(SESSION_WITHOUT_OWNER);
+            const isAuthenticated = accountService.isAuthenticated(
+                REQUEST_WITH_IS_AUTHENTICATED_TRUE
+            );
+            expect(isAuthenticated).toBe(true);
+        });
+    });
 });
```

Comment:
This should probably be improved. It's only testing the happy path, and I believe it's actually creating an invalid state, as you've created the account service with a "session without user" but isAuthenticated is truthy.

Reply:
well caught. mean to use `SESSION_WITH_OWNER`